windows/ReactNativeWebView/pch.h

```diff
@@ -1,6 +1,19 @@
 ﻿#pragma once
 
 #define NOMINMAX
+
+#if __has_include(<VersionMacros.h>)
+#include <VersionMacros.h>
+#else
+#define RNW_VERSION_AT_LEAST(x,y,z) false
+#endif
+
+#if RNW_VERSION_AT_LEAST(0,68,0)
+#define USE_WEBVIEW2 1
```

Comment:
rename to HAS_WEBVIEW2 (since you can still choose to use WebView 1 with a WinUI 2.x version that contains WV2)